com_languagepack/site/views/export/view.xml.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ * @package     Joomla.Site
+ * @subpackage  com_languagepack
+ *
+ * @copyright   Copyright (C) 2020 - 2020 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\MVC\View\HtmlView;
+
+/**
+ * Export view class
+ *
+ * @since  1.0
+ *
+ * @property-read   \Joomla\CMS\Document\XmlDocument $document
+ */
+class LanguagepackViewExport extends HtmlView
+{
+	/**
+	 * Execute and display a template script.
+	 *
+	 * @param   string  $tpl  The name of the template file to parse; automatically searches through the template paths.
+	 *
+	 * @return  string  A XML if successful, otherwise an XML Error.
+	 */
+	public function display($tpl = null)
+	{
+		/** @var LanguagepackModelExport $model */
+		$model = $this->getModel();
+
+		$exportData = $model->collectDataForExportRequest();
+
+		// Check for errors.
+		if (count($errors = $this->get('Errors')))
+		{
+			echo $this->renderXmlError($errors);
+
+			return;
+		}
+		elseif (!$exportData || !isset($exportData['total'])
+			|| !$exportData['total'])
+		{
+			echo $this->renderXmlError(['No Data Found']);
+
+			return;
+		}
+
+		$cmsVersion   = $model->getState(
+			$model->getName() . '.cms_version'
+		);
+		$languageCode = $model->getState(
+			$model->getName() . '.language_code'
+		);
+
+		// This document should always be downloaded
+		$this->document->setDownload(true);
```

Comment:
Not sure if this is the best choice... but okay I have commented this out on the  https://downloads.joomla.org and https://downloads2.sandbox.joomla.org/ for now...

Reply:
If you've commented it out on both sites - just comment it out in the main code for now. I normally install this component by file install on the main Downloads site rather than anything manual - so we need to be aware of loosing that setting :)